bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -148,6 +148,18 @@ export class Wallet {
         return this.methodHandler.getClient();
     }
 
+    /**
+     * Get chrysalis data.
+     */
+    // TODO: is this the correct type?
```

Comment:
is it?

Reply:
Is this resolved ?

Reply:
I think so, I changed the type and removed the comment

Reply:
I'm resolving this for you then.